src/test/units/DeploymentActions.test.js

```diff
@@ -14,7 +14,7 @@ import AppsActions from "../../js/actions/AppsActions";
 var server = config.localTestserverURI;
 config.apiURL = "http://" + server.address + ":" + server.port + "/";
 
-describe("scheduler migrations", function () {
+describe("DeploymentActions", function () {
```

Comment:
Why the file rename and move to `units`, though? We have specific scenario tests for `requestDeployments`, `revertDeployment`, `stopDeployment`. Imho `continueMigration.test.js` fits the scenarios better and is inline with the others.